src/core/test/fuzzing/src/test/scala/Fuzzing.scala

```diff
@@ -123,40 +175,48 @@ trait SerializationFuzzing[S <: PipelineStage with MLWritable] extends FuzzingMe
       val loadedStage = reader.load(path)
```

Comment:
maybe add assert after the save since we ignore the file not found exception now

Reply:
Done